### PR TITLE
vmui: remove redundant requests on the `Explore Cardinality` page

### PR DESCRIPTION
**Pull request by Loori-R**
### Describe Your Changes

Remove redundant requests on the Explore Cardinality page.

#6240

### Checklist

The following checks are **mandatory**:

- [ ] My change adheres [VictoriaMetrics contributing guidelines](https://docs.victoriametrics.com/contributing/).

### CODE REVIEW

**Review by dmitryk-dk — APPROVED** [review]
LGTM!

File: app/vmui/packages/vmui/src/pages/CardinalityPanel/hooks/useCardinalityFetch.ts [hunk 1]
```diff
@@ -72,23 +72,23 @@ export const useFetchQuery = (): {
...
+      date: dayjs(requestParams.date).subtract(1, "day").format(DATE_FORMAT),
     };
 
     const urls = [
       getCardinalityInfo(serverUrl, requestParams),
       getCardinalityInfo(serverUrl, prevDayParams),
     ];
 
-    if (prevDate !== date) {
+    if (prevDate !== date && totalParams.match || totalParams.focusLabel) {
```
Comment (hagen1778) [inline app/vmui/packages/vmui/src/pages/CardinalityPanel/hooks/useCardinalityFetch.ts:83]: could you please elaborate on this change? should it be `prevDate !== date && (totalParams.match || totalParams.focusLabel)` instead?

Comment (Loori-R) [reply app/vmui/packages/vmui/src/pages/CardinalityPanel/hooks/useCardinalityFetch.ts]: Thank you for noticing, there was a mistake here, I've corrected it.

- An additional request is needed to obtain the totals without filtering; this is necessary for `Total series` and `Percentage from total`.
- In cases where there is no filtering, the totals in both requests are identical, and an additional request can be avoided.

**Review by hagen1778 — APPROVED** [review]
LGTM

**Pull request merged**